src/planscape/plan/migrations/0010_remove_project_max_cost_project_max_budget_and_more.py

```diff
@@ -1,37 +1,79 @@
-# Generated by Django 4.1.3 on 2023-01-25 23:19
+# Generated by Django 4.1.3 on 2023-02-09 08:01
 
 from django.db import migrations, models
+import django.db.models.deletion
 
 
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('plan', '0009_conditionscores'),
```

Comment:
are all of these migration files relevant to this PR?

Reply:
kind of.
Changing conditions/migrations/0003_get_condition_stats.py meant changing all the migrations depending on it, which included everything beyond plan/migrations/0009*.